src/metadata_validator.py

```diff
@@ -356,8 +350,7 @@ def validate_required_props(self, data_record, msg_prefix):
                                 result[ERRORS].append(create_error("M028", [msg_prefix, ",".join([f'"{name}"' for name in self.program_names])], data_key, data_value))
                             else:
                                 data_record[PROPERTIES][data_key] = matched_val
-                    
-                    if node_type in ["study"] and data_key in ["study_name", "clinical_study_name"]:
+                    elif entity_type == "Study" and data_key in ["study_name", "clinical_study_name"]:
```

Comment:
Similar to "program_name", property name for study name should be configured in content.json.